Dockerfile

```diff
@@ -2,21 +2,13 @@ FROM quay.io/wikiwatershed/taudem:5.3.8
 
 MAINTAINER Azavea <systems@azavea.com>
 
-ENV RWD_VERSION 1.2.0
-
 COPY requirements.txt /tmp/
 COPY src/api/requirements.txt /tmp/src/api/
 RUN pip install --no-cache-dir -r /tmp/requirements.txt && \
     pip install --no-cache-dir -r /tmp/src/api/requirements.txt && \
     rm /tmp/requirements.txt && \
     rm /tmp/src/api/requirements.txt
 
-ENV PYTHONPATH :$PYTHONPATH
-
-COPY ./rwd_drb /usr/src/api/rwd_drb
-COPY ./rwd_nhd /usr/src/api/rwd_nhd
+ENV PYTHONPATH /usr/src:$PYTHONPATH
 
-COPY ./src/api /usr/src/api
 WORKDIR /usr/src/api
-
-ENTRYPOINT ["gunicorn", "-w", "4", "--log-syslog", "--bind", "0.0.0.0:5000", "main:app"]
```

Comment:
This fixes auto reloading in development, but the artifact that we deploy to quay.io still needs a copy of the source code to work.